test/cdk-test-integration/functions/functions.js

```diff
@@ -0,0 +1,19 @@
+var AWS = require('aws-sdk');
```

Comment:
Please choose a more descriptive name for this file. Maybe something like `put-item-function.js`?